CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
+* If you use any programming/scripting logic to list and select a minor version of Kubernetes before creating clusters with the `ListKubernetesVersions` API, note that starting from Kubernetes v1.27, the API returns `SupportPlan` as `[KubernetesOfficial, AKSLongTermSupport]`. Please ensure you update any logic to exclude `AKSLongTermSupport` versions to avoid any breaks and choose `KubernetesOfficial` support plan versions.  Otherwise, if LTS is indeed your path forward please first opt-into the Premium tier and the `AKSLongTermSupport` support plan versions from the `ListKubernetesVersions` API before creating clusters.  Refer [long term support](https://learn.microsoft.com//azure/aks/long-term-support) for more information.
+
+### Release Notes
+
+* Features:
+  * Generally Available - AKS supports [disabling Windows OutboundNAT](https://learn.microsoft.com/en-us/azure/aks/nat-gateway#disable-outboundnat-for-windows).
+  * Generally Available - [Automated Deployments](https://learn.microsoft.com/en-us/azure/aks/automated-deployments).
+  * Generally Available - [Security patch channel](https://learn.microsoft.com/en-us/azure/aks/auto-upgrade-node-os-image?tabs=azure-cli) for VHD updates.
+  * Public Preview - Deployment Safeguards now supports [mutations in Enforcement mode](https://learn.microsoft.com/en-us/azure/aks/deployment-safeguards). 
+  * Public Preview - [Native sidecars](https://learn.microsoft.com/en-us/azure/aks/istio-native-sidecar) under Istio-based service mesh addon in AKS.
+  * Public Preview - [AKS Automatic](https://learn.microsoft.com/en-us/azure/aks/intro-aks-automatic).
+  * Public Preview - [Node Initialization Taints](https://learn.microsoft.com/en-us/azure/aks/use-node-taints#use-node-initialization-taints-preview)
+ 
+* Behavioral Changes:
+  * Allow disabling NPM for existing clusters with "networkPolicy=none" for stable api version 2024-05-01. 
+  * Minimum supported k8s version for advancedNetworking is 1.21.0 and minimum supported k8s version for cilium clusters with advancedNetworking is 1.29.0.
+  * Disk Detach must complete before [node image auto upgrade](https://learn.microsoft.com/en-us/azure/aks/auto-upgrade-node-os-image) starts.
+  * Default network policy is "networkPolicy=none" when network policy is not set on new clusters starting from API version 2024-05-01.
+  * Block GetOsOptions operation in 2024-05-01/2024-05-02-preview API or later - [Github Issue 4261](https://github.com/Azure/AKS/issues/4261).
+  * [Advanced Container Netwrking Services](https://learn.microsoft.com/en-us/azure/aks/advanced-container-networking-services-overview) can only be enabled on cilium clusters with k8s v1.29.0 onwards.
```

Comment:
```suggestion
  * [Advanced Container Netwrking Services](https://learn.microsoft.com/azure/aks/advanced-container-networking-services-overview) can only be enabled on cilium clusters with k8s v1.29.0 onwards.
```
I'm not seeing this limitation detailed in the documentation you linked. Can you reach out to the owner to properly document this limitation?

Reply:
Advanced Container **Netwrking** Services -> Networking (spelling)